### PR TITLE
feat(review): sort the diffs by score desc

### LINKED ISSUE #85: Sort the screenshots by score DESC

### PR DESCRIPTION
**Pull request by oliviertassinari**
Closes #85.

### CODE REVIEW

File: src/server/routes/api.test.js [hunk 1]
```diff
@@ -83,10 +83,10 @@ describe('api routes', () => {
           .field('commit', 'test-commit')
           .field('branch', 'test-branch')
           .field('token', 'xx')
-          .expect(200)
           .expect((res) => {
             expect(res.body.id).not.toBeUndefined()
           })
+          .expect(200)
```
Comment (neoziro) [inline src/server/routes/api.test.js]: 👍

File: src/server/routes/www.test.js [hunk 2]
```diff
@@ -3,12 +3,106 @@ import { useDatabase } from 'server/testUtils'
...
 import ScreenshotBucket from 'server/models/ScreenshotBucket'
+import ScreenshotDiff from 'server/models/ScreenshotDiff'
 import www from './www'
 
 describe('GraphQL', () => {
   useDatabase()
 
+  describe('screenshotDiffs', () => {
+    it('should sort the diffs by score', async function () {
+      const organization = await Organization.query().insert({
```
Comment (neoziro) [inline src/server/routes/www.test.js]: We will have to rewrite everything using factories, it's simpler and more concise.

Comment (oliviertassinari) [inline src/server/routes/www.test.js]: Waiting for the PR to be merged.

Comment (neoziro) [inline src/server/routes/www.test.js]: @oliviertassinari done

**Comment by neoziro** [comment]
@oliviertassinari need rebase

**Pull request merged**